_posts/2016-12-04-encore.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: ENCORE ensemble similarity
+---
+
+The **ENCORE** ensemble similarity library has been integrated in the next
+version of MDAnalysis
+as
+[MDAnalysis.analysis.encore](http://www.mdanalysis.org/mdanalysis/documentation_pages/analysis/encore.html).
+It implements a variety of techniques for calculating similarities between
+structural ensembles (trajectories), as described in this publication:
```

Comment:
The bootstraping is also good to know. But that requires better docs that should explain what it does and where the limitations are.